pykintone/user_api/user_api_result.py

```diff
@@ -31,7 +31,9 @@ def __init__(self, response):
                 OrganizationTitle = namedtuple("OrganizationTitle", ["organization", "title"])
                 for ot in self.raw:
                     o = Organization.deserialize(ot["organization"])
-                    t = Title.deserialize(ot["title"])
+                    t = None
+                    if ot["title"] is not None:
```

Comment:
* `if "title" in ot:` is safer (It comes from my programming backbone, some language cause exception when evaluating the missing key(`ot["title"]`) ). 
* Are there the same case in `"organization"`? If it exists, I want to add the same guard as `"title"` for `"organization"`

Reply:
> `if "title" in ot:` is safer

I think this is not the right way because `ot` looks like `{'organization': {'code': '営業部_4pSdO2', 'description': '', 'name': '営業部', 'localNameLocale': 'ja', 'localName': '営業部_localName', 'parentCode': None, 'id': '5'}, 'title': None}`

The key "title" is in `ot` but the value is `None`.

Reply:
> Are there the same case in "organization"?

No. Possible combinations are:

| organization | job title |
|---------|----------|
| not belong | no |
| belong | no |
| belong | a title |